crates/ra_assists/src/assists/add_derive.rs

```diff
@@ -9,6 +7,22 @@ use ra_syntax::{
 
 use crate::{Assist, AssistCtx, AssistId};
 
+// Assist: add_derive
```

Comment:
Why not a doc comment. That would be useful when using rustdoc in document private items mode.

Reply:
If using doc comments, it would also be useful to place the description as first line, instead of name, as the first line is used by rustdoc aa summary.

Reply:
Gentle ping @matklad. Did you miss this?

Reply:
Indeed, I've missed this. 

There are two reasons for not making this a doc comment:

* I don't want rustdoc treating snippets as tests. I can ignore test with `ignore`, but that still prints the `ignored` line. I can also `` ```text ``, but that just look weird. 
* In theory, it's possible that a single function produces several assists, in which case a doc comment wouldn't work.

Reply:
Although maybe we should go in the opposite direction and indeed *require* that each assists gots a separate function?

That way, we can auto-generate the "index of all assists" *module* as well...

Reply:
+1